client/my-sites/domains/controller.jsx

```diff
@@ -30,8 +30,7 @@ import { getCurrentUser } from 'state/current-user/selectors';
  */
 const productsList = productsFactory();
 
-module.exports = {
-
+export default {
```

Comment:
It's being minified anyway. Don't think it matters, and can do it once I decide to kill the `productsFactory()` if you think it's better that way.